test/models/report_test.rb

```diff
@@ -3,7 +3,70 @@
 require 'test_helper'
 
 class ReportTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  setup do
+    @report1 = FactoryBot.create(:report)
+    @report2 = FactoryBot.create(:report)
+    @report3 = FactoryBot.create(:report)
+    @report4 = FactoryBot.create(:report)
+  end
+
+  test '#editable? should return true when target user tries to edit a report created by target user' do
+    target_user = FactoryBot.create(:user)
+    report = FactoryBot.create(:report, user: target_user)
+
+    assert report.editable?(target_user)
+  end
+
+  test '#editable? should return false when target user tries to edit a report created by another user' do
+    target_user = FactoryBot.create(:user)
+    another_user = FactoryBot.create(:another_user)
+    report = FactoryBot.create(:report, user: another_user)
+
+    assert_not report.editable?(target_user)
+  end
+
+  test '#created_on should convert created_at that is attribute of Report to Date object' do
+    report = FactoryBot.create(:report, created_at: Time.zone.parse('2023-01-01'))
+
+    travel_to Date.new(2023, 1, 1) do
+      assert_equal Time.zone.today, report.created_on
+    end
```

Comment:
わざわざtravel_toを使わなくてもこれでいいと思います。

```suggestion
    assert_equal Date.new(2023, 1, 1), report.created_on
```